components/wb-fieldflow/fieldflow.js

```diff
@@ -134,7 +134,10 @@ var componentName = "wb-fieldflow",
 			// Transform the list into a select, use the first paragrap content for the label, and extract for i18n the name of the button action.
 			var bodyID = wb.getId(),
 				stdOut,
-				formElm, $form;
+				formElm,
+				$form,
+				btnColor = config.btnColor && [ "default", "primary", "success", "info", "warning", "danger", "link" ].indexOf( config.btnColor ) >= 0 ? config.btnColor : "default",
```

Comment:
```suggestion
				btnType = config.btnType && [ "default", "primary", "success", "info", "warning", "danger", "link" ].indexOf( config.btnType ) >= 0 ? config.btnType : "default",
```
A bit of word-smithing-nitpicky. It feels like we're not really the background colour of the button but the type of button used (like the link type doesn't have a background color)

Reply:
or btnStyle

Reply:
Done. Thanks